packages/cli/src/api/formats/po.test.js

```diff
@@ -71,7 +89,59 @@ describe("pofile format", function() {
     })
 
     const filename = path.join("locale", "en", "messages.po")
-    expect(format.read(filename)).toMatchSnapshot()
+    const actual = format.read(filename)
+    mockFs.restore()
+    expect(actual).toMatchSnapshot()
+  })
+
+  it("should correct badly used comments", function() {
+    const po = PO.parse(`
+      #. First description
+      #. Second comment
+      #. Third comment
+      msgid "withMultipleDescriptions"
+      msgstr "Extra comments are separated from the first description line"
+
+      # Translator comment
+      #. Single description only
+      #. Second description?
+      msgid "withDescriptionAndComments"
+      msgstr "Second description joins translator comments"
+    `)
+
+    mockFs({
+      locale: {
+        en: {
+          "messages.po": po.toString()
+        }
+      }
+    })
+
+    const filename = path.join("locale", "en", "messages.po")
+    const actual = format.read(filename)
+    mockFs.restore()
+    expect(actual).toMatchSnapshot()
+  })
+
+  it("should throw away additional msgstr if present", function() {
```

Comment:
We could use `mockConsole` here, at least for hiding that error test output.

https://github.com/lingui/js-lingui/blob/072afd8d3664dce514c0628f7e77eef207682ce6/packages/cli/src/mocks.js#L12

Reply:
Yea, that's probably good idea, I was running only tests for PO and haven't really considered it might be annoying when running everything :) Will fix that tomorrow.

Reply:
There're few hanging console warnings in output, but I'm trying to limit it as much as possible. As a bonus, you can also test the error message ;) But as I said, that's just a nitpick. Hiding console output is more important.